modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationQueueEntryLocalServiceTest.java

```diff
@@ -147,6 +171,77 @@ public void testAddNotificationQueueEntry() throws Exception {
 			notificationQueueEntry);
 	}
 
+	@Test
+	public void testDeleteNotificationQueueEntries() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, "Name", "name")));
+
+		NotificationContext notificationContext =
+			NotificationTemplateUtil.createNotificationContext(
+				TestPropsValues.getUser(), StringUtil.randomString(),
+				StringUtil.randomString(),
+				Arrays.asList(
+					NotificationRecipientSettingUtil.
+						createNotificationRecipientSetting(
+							"from", "[%CURRENT_USER_EMAIL_ADDRESS%]"),
+					NotificationRecipientSettingUtil.
+						createNotificationRecipientSetting(
+							"fromName",
+							Collections.singletonMap(
+								LocaleUtil.US, RandomTestUtil.randomString())),
```

Comment:
@fabioafreitas you can use `LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())` here